# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/xul/XULFormControlAccessible.h
# Commit: 5d76535b47fc
# Full Hash: 5d76535b47fced08e46753a0262211732fb26417
# Author: Eitan Isaacson <eitan@monotonous.org>
# Date: 2022-01-05 09:33:53
# Regressor Bug: 1737944
# File Overlap Count: 2
# Description:
#   Bug 1737944 - P7: Move and rename GetPositionAndSizeInternal to Accessible. r=Jamie
#   
#   Now that the AccGroupInfo getter is in Accessible, the position and size
#   getter could go there too.
#   
# ==============================================================================

diff -r fca1ff7eabf4 -r 5d76535b47fc accessible/xul/XULFormControlAccessible.h
--- a/accessible/xul/XULFormControlAccessible.h	Tue Jan 04 21:01:36 2022 +0000
+++ b/accessible/xul/XULFormControlAccessible.h	Tue Jan 04 21:01:37 2022 +0000
@@ -141,15 +141,16 @@
  public:
   XULToolbarButtonAccessible(nsIContent* aContent, DocAccessible* aDoc);
 
-  // LocalAccessible
-  virtual void GetPositionAndSizeInternal(int32_t* aPosInSet,
-                                          int32_t* aSetSize) override;
-
   // nsXULToolbarButtonAccessible
   static bool IsSeparator(LocalAccessible* aAccessible);
 
   // Widgets
   virtual bool IsAcceptableChild(nsIContent* aEl) const override;
+
+ protected:
+  // LocalAccessible
+  virtual void GetPositionAndSetSize(int32_t* aPosInSet,
+                                     int32_t* aSetSize) override;
 };
 
 /**
